### DISCUSSION TITLE
Gamemode/mutator idea: Gun Game/Gun Master

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1189] [Ideas and Feature Requests]
Since there hasn't been any real new gamemode/mutator for quite some time, i figured we could use something basic like Gun Game, as it doesn't need any new assets to be made for it.
Basically what happens is:

Everyone spawns with the same weapon, and every 2 kills with it, you go to the next tier, 2 more kills with that gun, next tier, until you end up with the one hardest to kill, and if you kill 2 with that one, you win the match.

On RE terms, this order could be, in theory : Rifle > Zapper >  Plasma > Flamer > SMG > Shotgun > Sword > Pistol > Claw (could be only available for this gamemode ).

On the scoreboard, you could see who is closer to the "top tier", instead of points, or even a bar of progress, on the side of the screen.

It's a basic and fun gamemode that forces you to adapt and not get too comfortable with only using one weapon for the duration of the match.

Some examples of how this plays :

https://www.youtube.com/watch?v=OzIRCEJmCkc 
https://www.youtube.com/watch?v=HquIBIWCiqU

So, what do you think?

**Comment by dc-redeclipse** [comment]
Nice idea :D
